microsoft-edge/webview2/how-to/publish-uwp-app-store.md

```diff
@@ -0,0 +1,139 @@
+
+1. Note where the bundle was placed (also in the build window) and **Cancel** this dialog box.
+
+1. Use the generated app package to install the app on your machine.  Look for any oddities regarding required packages and functionality.
+
+1. If possible, install your app on other machines also.  Look for any oddities regarding required packages and functionality.
+
+
+<!-- ====================================================================== -->
+## Step 5: Run Windows App Certification Kit (WACK)
+
+Once you're satisfied that your packaged app works, run the Windows App Certification Kit (WACK):
+
+1. Generate the app package again, as above.  That's the easiest way to get to the Windows App Certification Kit (WACK) process.
+
+1. Once the package creation is complete, select **Launch Windows App Certification Kit**.
+
+1. You must approve the User Access Control (UAC) dialog, for the Windows App Certification Kit (WACK) to run correctly.
+
+1. Select **All** for the test suite, and then click **Next**.  The validation will take a few minutes.
+
+1. Once complete, the Windows App Certification Kit (WACK) will show a results page.  If the app failed, click the link to review the results.
+
+
+### Resolving tests
+
+The results page of the Windows App Certification Kit (WACK) app shows any tests that need to be resolved.
+
+
+#### Restricted name-space
+
+If your app failed the **Restricted name-space** check, edit the `package.appxmanifest` file and remove the `restrictedcapabilities` line and any capabilities that use `rescap`.
+
+WebView2 shouldn't require any restricted capabilities, to operate.  If your app needs restricted capabilities, you'll need to either ask for a store exception through [Partner Center](https://partner.microsoft.com), or only distribute the app through non-Store avenues.
+
+
+#### Branding
+
+If your app failed the **Branding** checks, update all the Visual Assets in the manifest as described above.
+
+
+#### App Capabilities
+
+If your app failed the **App Capabilities** tests, turn off any unneeded capabilities.  If you need a special use capability you will need to get Microsoft Store approval or use non-Store distribution methods.
+
+
+#### General metadata correctness
+
+If your app failed the **General meta data correctness** tests with an error in `IPopup4`, update the **microsoft.UI.xaml** NuGet package to version **Microsoft.UI.Xaml.2.8.0-prerelease.220118001** or higher.  There was a bug in the manifests, prior to that version.
+
+
+#### Supported API
+
+<!--
+which APIs/which SDK is being tested?
+This check tests whether the XYZ APIs that are used by your app are supported by ABC.
+-->
+
+If your app failed the **Supported API** test, update the <!--which SDK, XAML? WebView2?-->SDK to version 22000 <!--correct specific full number?--> or higher, to get the latest version of the allowed-APIs list.  There should be no blocking APIs for WebView2 in the current <!-- which SDK? -->SDK.
```

Comment:
@DavidShoe 
If your app failed the Supported API test, update the [which SDK, XAML? WebView2?]SDK to version 22000 [correct specific full number?] or higher, to get the latest version of the allowed-APIs list. There should be no blocking APIs for WebView2 in the current [which SDK?]SDK.

@nishitha-burman